code/modules/cargo/exports/weapons.dm

```diff
@@ -4,85 +4,85 @@
 	include_subtypes = FALSE
 
 /datum/export/weapon/makeshift_shield
-	cost = 30
+	cost = 250
 	unit_name = "unknown shield"
 	export_types = list(/obj/item/shield/riot, /obj/item/shield/riot/roman, /obj/item/shield/riot/buckler, /obj/item/shield/makeshift)
 
 /datum/export/weapon/riot_shield
-	cost = 50
+	cost = 250
 	unit_name = "riot shield"
 	export_types = list(/obj/item/shield/riot, /obj/item/shield/riot/tower)
 
 /datum/export/weapon/riot_shield
-	cost = 70
+	cost = 350
 	unit_name = "flash shield"
 	export_types = list(/obj/item/assembly/flash/shield)
 
 /datum/export/weapon/tele_shield
-	cost = 100
+	cost = 650
 	unit_name = "tele shield"
 	export_types = list(/obj/item/shield/riot/tele, /obj/item/shield/energy)
 
 /datum/export/weapon/baton
-	cost = 100
+	cost = 850
 	unit_name = "stun baton"
 	export_types = list(/obj/item/melee/baton)
 	exclude_types = list(/obj/item/melee/baton/cattleprod)
 	include_subtypes = TRUE
 
 /datum/export/weapon/knife
-	cost = 100
+	cost = 950
 	unit_name = "combat knife"
 	export_types = list(/obj/item/kitchen/knife/combat)
 
 /datum/export/weapon/taser
-	cost = 200
+	cost = 500
 	unit_name = "advanced taser"
 	export_types = list(/obj/item/gun/energy/e_gun/advtaser)
 
 /datum/export/weapon/laser
-	cost = 200
+	cost = 800
 	unit_name = "laser gun"
 	export_types = list(/obj/item/gun/energy/laser)
 
 /datum/export/weapon/disabler
-	cost = 50
+	cost = 300
 	unit_name = "disabler"
 	export_types = list(/obj/item/gun/energy/disabler)
 
 /datum/export/weapon/energy_gun
-	cost = 200
+	cost = 500
 	unit_name = "energy gun"
 	export_types = list(/obj/item/gun/energy/e_gun)
 
 /datum/export/weapon/wt550
-	cost = 130
+	cost = 1250
 	unit_name = "WT-550 automatic rifle"
 	export_types = list(/obj/item/gun/ballistic/automatic/wt550)
 
 /datum/export/weapon/shotgun
-	cost = 200
+	cost = 4000
 	unit_name = "combat shotgun"
 	export_types = list(/obj/item/gun/ballistic/shotgun/automatic/combat)
 
 /datum/export/weapon/flashbang
-	cost = 5
+	cost = 50
 	unit_name = "flashbang grenade"
 	export_types = list(/obj/item/grenade/flashbang)
 
 /datum/export/weapon/teargas
-	cost = 5
+	cost = 50
 	unit_name = "tear gas grenade"
 	export_types = list(/obj/item/grenade/chem_grenade/teargas)
 
 /datum/export/weapon/flash
-	cost = 5
+	cost = 50
```

Comment:
This might become a powerful AFK meta with exosuit printers.

Reply:
Will lower to 25.